docs/modules/plugins/partials/common-filters.adoc

```diff
@@ -5,3 +5,7 @@
 |textPart
 |Filter elements by their text parts.
 |tagName(h3)->filter.textPart(Welcome)
+
+|attribute
+|Filter elements by their attribute values
+|tagName(input)->filter.attribute(type=button) or tagName(input)->filter.attribute(disabled)
```

Comment:
please describe the difference:
`filter.attribute(type=button)`
vs
`filter.attribute(disabled)`
vs
`filter.attribute(type=)`